avaje-jex/src/main/java/io/avaje/jex/http/sse/SseHandler.java

```diff
@@ -0,0 +1,43 @@
+package io.avaje.jex.http.sse;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.function.Consumer;
+
+import io.avaje.jex.core.Constants;
+import io.avaje.jex.http.BadRequestException;
+import io.avaje.jex.http.Context;
+import io.avaje.jex.http.ExchangeHandler;
+
+/** Handler that configures a request for Server Sent Events */
+public class SseHandler implements ExchangeHandler {
```

Comment:
This looks like it could be internal / not public?

Reply:
If it's package private it couldn't be accessible to the routing methods